pkgs/by-name/pt/pterm/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, gcc11Stdenv
```

Comment:
is gcc11 really needed?

Reply:
Using `stdenv` it fails to build, the error seems like a simple enough issue to fix. I'll raise it upstream and see what I can hear back:
```
pterm> gcc -O2 -g2 `sdl-config --cflags`      -DHOST_Linux  -std=gnu99 -Wall -Werror -Wno-format -c dtnetsubs.c
pterm> dtnetsubs.c: In function 'dtSend':
pterm> dtnetsubs.c:868:21: error: the comparison will always evaluate as 'true' for the address of 'ssem' will never be NULL [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Waddress-Werror=address8;;]
pterm>   868 |     if (ssemp (fet) != NULL)
pterm>       |                     ^~
pterm> In file included from dtnetsubs.c:27:
pterm> types.h:255:17: note: 'ssem' declared here
pterm>   255 |     sem_t       ssem;                   /* For waking send thread */
pterm>       |                 ^~~~
pterm> cc1: all warnings being treated as errors
pterm> make: *** [Makefile.common:50: dtnetsubs.o] Error 1
error: builder for '/nix/store/kjn46n6d637w24g1cgnsgw556lg4c956-pterm-6.0.4.drv' failed with exit code 2;
       last 10 log lines:
       > dtnetsubs.c: In function 'dtSend':
       > dtnetsubs.c:868:21: error: the comparison will always evaluate as 'true' for the address of 'ssem' will never be NULL [-Werror=address]
       >   868 |     if (ssemp (fet) != NULL)
       >       |                     ^~
       > In file included from dtnetsubs.c:27:
       > types.h:255:17: note: 'ssem' declared here
       >   255 |     sem_t       ssem;                   /* For waking send thread */
       >       |                 ^~~~
       > cc1: all warnings being treated as errors
       > make: *** [Makefile.common:50: dtnetsubs.o] Error 1
       For full logs, run 'nix log /nix/store/kjn46n6d637w24g1cgnsgw556lg4c956-pterm-6.0.4.drv'.
```

Reply:
@eclairevoyant discussed the error with one of the developers of pterm and he recommended the following patch.

Going to work on making sure that gets upstreamed to pterm but Ive confirmed the binary is working as intended post-patch.